networking/openshift_sdn/migrate-to-openshift-sdn.adoc

```diff
@@ -21,7 +21,7 @@ include::modules/nw-ovn-kubernetes-rollback.adoc[leveloffset=+1]
 
 * xref:../../networking/cluster-network-operator.adoc#nw-operator-configuration-parameters-for-openshift-sdn_cluster-network-operator[Configuration parameters for the OpenShift SDN network plugin]
 * xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd]
-* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy]
```

Comment:
I could not click-check this link because this assembly does not appear in any topic maps. This is just a heads-up, but I wonder if we should raise the presence of an orphan assembly with Kathryn, please ping me on Slack if I should do it (no need to hold up this PR over it)

Reply:
Do you mean the xref assembly or the assembly that this xref is in?